google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudLibraryProjectState.java

```diff
@@ -139,9 +139,13 @@ void syncCloudLibrariesBom() {
   }
 
   private Optional<String> loadCloudLibraryBomVersion(Module module) {
-    return loadCloudLibraryBom(module)
-        .map(MavenDomDependency::getVersion)
-        .map(GenericValue::getStringValue);
+    return ApplicationManager.getApplication()
+        .runReadAction(
```

Comment:
Just out of curiosity and to be sure how to use this myself, what is the general purpose of using `runReadAction` here?